public/sass/partials/_download-app.scss

```diff
@@ -0,0 +1,66 @@
+.download-app-section {
+	margin-top: 2em;
+	padding-bottom: 2em;
+	@include flexbox;
+	position: relative;
+	.download-img {
+		position: relative;
+		margin-top: 5%;
+		margin-right: 5%;
+		margin-left: 5%;
+		width: 16em;
+		img:nth-child(1) {
+			z-index: 0;
+			height: 23em;
+		}
+		img:nth-child(2) {
+			position: absolute;
+			width: 78%;
+			top: -5%;
+			left: 33%;
+			z-index: 0;
+		}
+		@media screen and (max-width:1075px) {
+			display: none;
+		}
+	}
+		p {
+			font: $text-light-font;
+			color: $color-text-medium;
+			margin-top: 3em;
+	}
+	.download-link {
+		margin-top: 3em;
+		margin-right: .5em;
+		margin-left: .5em;
+		}
+		h1 {
+			font: $head-big-light-font;
+			span {
+				font-weight: 600;
+			}
+			span.accent-bar-L {
+				width: 13%;
+				color: $color-5;
+			}
+		}
+	.download-link-wrapper {
```

Comment:
Again, so much code nested under the parent. These can be root selectors and only nested if there is a dependency on the part for a modification.